app/components/NewCredentialForm.tsx

```diff
@@ -0,0 +1,86 @@
+/*!
+ * Copyright © 2022 United States Government as represented by the Administrator
+ * of the National Aeronautics and Space Administration. No copyright is claimed
+ * in the United States under Title 17, U.S. Code. All Other Rights Reserved.
+ *
+ * SPDX-License-Identifier: NASA-1.3
+ */
+
+import type { DataFunctionArgs } from '@remix-run/node'
+import { Form, Link, useLoaderData } from '@remix-run/react'
+import { Button, Dropdown, Label, TextInput } from '@trussworks/react-uswds'
+import { useState } from 'react'
+import ReCAPTCHA from 'react-google-recaptcha'
+import { getEnvOrDieInProduction } from '~/lib/env'
+import { ClientCredentialVendingMachine } from '~/routes/user/client_credentials.server'
+
+export async function loader({ request }: DataFunctionArgs) {
+  const machine = await ClientCredentialVendingMachine.create(request)
+  const client_credentials = await machine.getClientCredentials()
+  const groups = machine.groups
+  const recaptchaSiteKey = getEnvOrDieInProduction('RECAPTCHA_SITE_KEY')
+  return { client_credentials, recaptchaSiteKey, groups }
+}
+export function NewCredentialForm() {
```

Comment:
Add newline between functions:

```suggestion

export function NewCredentialForm() {
```